app/react/Library/components/specs/SortButtons.spec.js

```diff
@@ -110,7 +110,7 @@ describe('SortButtons', () => {
             .find('span')
             .last()
             .text()
-        ).toBe('Search relevance');
+        ).toBe('Date modified (Recently)');
```

Comment:
@grafitto the description and string being tested do not match.
i think the default still should be "Search Relevance" when there is an active search term.
what you are testing here its already being tested by the full component snapshot in a previous test, this is something i think we should avoid. refactor the test to not use snapshots would be a good option.

Reply:
The "Date Modified" indeed is being tested by the snapshot.  The "Search relevance", on the other hand, is not, and is only tested here.  The issue is that this is testing the existence of "Search relevance" by position, and it is now on a different location.

The test has been reverted to look for Search relevance but on the correct position.  Still, it would be interesting to make this test resilient to position changes of the element, since there is nothing important regarding the position of the element.  It just needs to be there.

This is a good exercise in writing the "right" tests: the position was not important, the text and its link are.  The text is being tested, the link isn't and the position is being tested when it is not important.